crates/client/src/client/schema/chain.rs

```diff
@@ -264,6 +265,7 @@ impl From<ConsensusParameters> for fuel_core_types::fuel_tx::ConsensusParameters
             fee_params: params.fee_params.into(),
             chain_id: params.chain_id.0.into(),
             gas_costs: params.gas_costs.into(),
+            base_asset_id: AssetId::default().into(),
```

Comment:
Why does one version of `ConsensusParameters` have the `base_asset_id` field and another doesn't?

Also, Idk how I feel about using `default` here. It kinda feels arbitrary.

Reply:
`ConsensusParameters` should always have `base_asset_id` now. I might be missing something - can you point me to where you're seeing a version of `ConsensusParameters` that doesn't?

Also, using `default` here is temporary, since the base asset ID configuration isn't done for `fuel-core` - it's currently in progress here: https://github.com/FuelLabs/fuel-core/pull/1339. I had to update the `fuel-vm` side, including the `fuel-tx::ConsensusParameters` struct, to include the base asset ID, so this is the non-parameterized version in the meantime. This will be configured by the `params` once that is updated.

Reply:
Oh. Sorry. I meant to ask why you aren't using `params.base_asset_id` and you're manually initializing one. If this is an intermediate state then I won't worry about it.